src/main.c

```diff
@@ -454,6 +466,80 @@ int main(int argc, char *argv[])
 #endif /* MAIKO_ENABLE_ETHERNET */
 
     }
+
+#ifdef MAIKO_ENABLE_NETHUB
+    else if (!strcmp(argv[i], "-nh-host")) {
+      if (argc > ++i) {
+        setNethubHost(argv[i]);
+      } else {
+        fprintf(stderr, "Missing argument after -nh-host\n");
+        exit(1);
+      }
+    }
+    else if (!strcmp(argv[i], "-nh-port")) {
+      if (argc > ++i) {
+        errno = 0;
+        tmpint = strtol(argv[i], (char **)NULL, 10);
+        if (errno == 0 && tmpint > 0) {
+          setNethubPort(tmpint);
+        } else {
+          fprintf(stderr, "Bad value for -nh-port\n");
+          exit(1);
+        }
+      } else {
+        fprintf(stderr, "Missing argument after -nh-port\n");
+        exit(1);
+      }
+    }
+    else if (!strcmp(argv[i], "-nh-mac")) {
+      if (argc > ++i) {
+        int b0, b1, b2, b3, b4, b5;
+        if (sscanf(argv[i], "%x-%x-%x-%x-%x-%x",  &b0, &b1, &b2, &b3, &b4, &b5) == 6) {
+          setNethubMac(b0, b1, b2, b3, b4, b5);
+        } else {
+          fprintf(stderr, "Invalid argument for -nh-mac\n");
+          exit(1);
+        }
+      } else {
+        fprintf(stderr, "Missing argument after -nh-mac\n");
+        exit(1);
+      }
+    }
+    else if (!strcmp(argv[i], "-nh-loglevel")) {
+      if (argc > ++i) {
+        errno = 0;
+        tmpint = strtol(argv[i], (char **)NULL, 10);
+        if (errno == 0 && tmpint >= 0) {
+          setNethubLogLevel(tmpint);
+        } else {
+          fprintf(stderr, "Bad value for -nh-loglevel\n");
+          exit(1);
+        }
+      } else {
+        fprintf(stderr, "Missing argument after -nh-loglevel\n");
+        exit(1);
+      }
+    }
+#endif /* MAIKO_ENABLE_NETHUB */
+
+#if defined(MAIKO_EMULATE_TIMER_INTERRUPTS) || defined(MAIKO_EMULATE_ASYNC_INTERRUPTS)
+    else if (!strcmp(argv[i], "-intr-emu-insns")) {
+      if (argc > ++i) {
+        errno = 0;
+        tmpint = strtol(argv[i], (char **)NULL, 10);
+        if (errno == 0 && tmpint > 1000) {
+          timerAsyncEmulationInsnsCountdown = tmpint;
```

Comment:
Not a big fan of emu-insns -- I don't see the value in identifying the counter as being in "emulated instructions".

Reply:
done (the naming was meant to be read as timerAsyncEmulation-InsnsCountdown, not timerAsync-EmulationInsns-Countdown; the name is now `insnsCountdownForTimerAsyncEmulation`)